awscli/testutils.py

```diff
@@ -57,6 +57,7 @@
 from awscli.plugin import load_plugins
 from awscli.clidriver import CLIDriver
 from awscli import EnvironmentVariables
+import awscli.errorhandler
```

Comment:
Did not know that plan. OK then. The next PR will unregister the errorhandler like this.

        client.meta.events.unregister(
            'after-call', unique_id='awscli-error-handler')

This line will not be as obvious as the `import awscli.errorhandler`, though. Which means we could probably forget to also remove this line, when we eventually remove the `errorhandler` module entirely. That being said, an extra `unregister('event', unique_id='an_nonexist_id')` will not bring substantial harm anyway.